templates/Dockerfile_ui2.template

```diff
@@ -1,32 +1,37 @@
-FROM ubuntu:16.04 as builder
+FROM ubuntu:18.04 as builder
 
-RUN apt-get update \
-    && apt-get upgrade -y \
-    && apt-get install -y \
+RUN apt update \
+    && apt install -yq \
+    openjdk-11-jre \
```

Comment:
I just stumbled across this the other day for the first time, so passing it along.... Java is only needed for build time, isn't it, for the Swagger code-gen? Ideally you would remove this after building.

However, you need to remove it on the [same layer you added it ](https://blog.replicated.com/refactoring-a-dockerfile-for-image-size/), so it would be some ugly looking code. Plus we are hopefully going to get rid of the UI container entirely anyway. And finally, I don't know that we care that much about the image/container size in this case.

Still, something to ponder for future PRs.

Reply:
This entire container is only needed for build time

Reply:
To elaborate, this is already a https://docs.docker.com/develop/develop-images/multistage-build/ since 1.5.4 I think

Reply:
Oh, thanks for clarifying.